src/clients/FlexbaseClient.Doc.ts

```diff
@@ -0,0 +1,14 @@
+import { FlexbaseClientBase } from './FlexbaseClient.Base';
+
+export class FlexbaseClientDoc extends FlexbaseClientBase {
+  async getImage(docId: string): Promise<ArrayBuffer | null> {
```

Comment:
would `getDocument` be a better name?

Reply:
maybe `getDocumentImage`? I know there's an endpoint to get a PDF document, so we can use`getDocument` name when we create this call (?)

Reply:
Works for me